src/resources/AWS/AWS.ts

```diff
@@ -5,6 +5,8 @@ export default class AWS extends Resource {
     static baseUrl = `/rest/aws/instances/agents`;
 
     list() {
```

Comment:
Could we still pass another org id as query string if needed?
```suggestion
    list(options: {organizationId: string} = {organizationId: this.api.organizationId}) {
```